From b53eb3d7d775eac0cd991ce8c31e7214571b526e Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:10:40 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 459b03562ca8744798e1e02a3a610fc4c93b8ba1
Merge: 2a280339 e109529d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 20 23:56:22 2018 +0000

    Snap for 4905103 from e109529dc07e2dd9fe9a49c39a347a2e478495a1 to pi-dr1-release

    Change-Id: Ifb9e4484be42149d9c142bbc1c0977a3a7ad2af7

commit e109529dc07e2dd9fe9a49c39a347a2e478495a1
Author: Bill Yi <byi@google.com>
Date:   Fri Jul 20 11:38:48 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I5a4a3da65d7fd382a9f4d20c444cd4a6b41c5c00

commit 2a280339ee5741430e7427aaf0e012d94a63e1c8
Merge: 12d33e14 ead0509d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:00:55 2018 +0000

    Snap for 4859140 from ead0509d6d5f7b8d2b586f3b2d0ac5f6e0b3192a to pi-dr1-release

    Change-Id: I8b9dcf0d3f5d66f7acb0ebfdfe51df8591de7d2f

commit ead0509d6d5f7b8d2b586f3b2d0ac5f6e0b3192a
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 07:43:21 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ief3ab68deb4170e02765e0576128766e181cbdfb

commit 12d33e144c340d3ba821c08a72aa13573dd45557
Merge: f79487cf 259cd820
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 6 03:05:16 2018 +0000

    Snap for 4823496 from 259cd82090bd785d8182c48627e70c8c544ac7f1 to pi-dr1-release

    Change-Id: I72e351eaee04f4025de4dddc171c97b4f60fb745

commit f79487cf62985f3f6f6c5e113317c5ab260879e8
Merge: 21b3c47a e428613e
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:07:57 2018 +0000

    Snap for 4820872 from e428613eb8cadc059a1631027cc72172935fc3a3 to pi-dr1-release

    Change-Id: I6b82d8cec147a4c0ec23042cc9fbab6273ebfc09

Change-Id: Idd86c48a4e7d84e8c6df4dcc18909dec40074659
---
 res/values-hy/strings.xml | 2 +-
 res/values-iw/strings.xml | 4 ++--
 res/values-tl/strings.xml | 2 +-
 res/values-uk/strings.xml | 2 +-
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/res/values-hy/strings.xml b/res/values-hy/strings.xml
index f6aecd77..af876fa8 100644
--- a/res/values-hy/strings.xml
+++ b/res/values-hy/strings.xml
@@ -119,7 +119,7 @@
     <string name="permission_title" msgid="6495415273398916381">"Թույլտվություններ (<xliff:g id="PERMISSION">%1$s</xliff:g>)"</string>
     <string name="no_apps" msgid="1965493419005012569">"Հավելվածներ չկան"</string>
     <string name="location_settings" msgid="1774875730854491297">"Տեղորոշման կարգավորումներ"</string>
-    <string name="location_warning" msgid="8778701356292735971">"Այս սարքի տեղադրության ծառայությունները տրամադրում է <xliff:g id="APP_NAME">%1$s</xliff:g> հավելվածը: Տեղադրության ցուցադրման կարգավորումները կարող եք փոխել տեղադրության կարգավորումներից:"</string>
+    <string name="location_warning" msgid="8778701356292735971">"Այս սարքի տեղորոշման ծառայությունները տրամադրում է <xliff:g id="APP_NAME">%1$s</xliff:g> հավելվածը: Տեղադրության ցուցադրման կարգավորումները կարող եք փոխել տեղադրության կարգավորումներից:"</string>
     <string name="system_warning" msgid="7103819124542305179">"Եթե չտրամադրեք այս թույլտվությունը, ձեր սարքի հիմնական գործառույթները հնարավոր է սխալ աշխատեն:"</string>
     <string name="permission_summary_enforced_by_policy" msgid="3418617316188986205">"Սահմանված է կանոններով"</string>
     <string name="loading" msgid="7811651799620593731">"Բեռնում…"</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index a7d2dc51..163df2d9 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -92,11 +92,11 @@
     <string name="devicePerms" msgid="6733560207731294504">"גישה למכשיר"</string>
     <string name="no_new_perms" msgid="6657813692169565975">"עדכון זה לא דורש הרשאות חדשות."</string>
     <string name="grant_confirm_question" msgid="4690289297029223742">"האם ברצונך להעניק את ההרשאות הבאות? תוענק גישה אל:"</string>
-    <string name="grant_dialog_button_deny" msgid="2176510645406614340">"לא, אין מצב"</string>
+    <string name="grant_dialog_button_deny" msgid="2176510645406614340">"לא, תודה"</string>
     <string name="grant_dialog_button_more_info" msgid="2218220771432058426">"למידע נוסף"</string>
     <string name="grant_dialog_button_deny_anyway" msgid="847960499284125250">"לדחות בכל מקרה"</string>
     <string name="current_permission_template" msgid="6378304249516652817">"<xliff:g id="CURRENT_PERMISSION_INDEX">%1$s</xliff:g> מתוך <xliff:g id="PERMISSION_COUNT">%2$s</xliff:g>"</string>
-    <string name="permission_warning_template" msgid="7332275268559121742">"‏האם לאשר ל&lt;b&gt;<xliff:g id="APP_NAME">%1$s</xliff:g>&lt;/b&gt; לבצע <xliff:g id="ACTION">%2$s</xliff:g>?"</string>
+    <string name="permission_warning_template" msgid="7332275268559121742">"‏האם לאשר ל&lt;b&gt;<xliff:g id="APP_NAME">%1$s</xliff:g>&lt;/b&gt; <xliff:g id="ACTION">%2$s</xliff:g>?"</string>
     <string name="permission_revoked_count" msgid="7386129423432613024">"<xliff:g id="COUNT">%1$d</xliff:g> הרשאות מושבתות"</string>
     <string name="permission_revoked_all" msgid="8595742638132863678">"כל ההרשאות מושבתות"</string>
     <string name="permission_revoked_none" msgid="2059511550181271342">"אין הרשאות מושבתות"</string>
diff --git a/res/values-tl/strings.xml b/res/values-tl/strings.xml
index 2b19ad78..3b168904 100644
--- a/res/values-tl/strings.xml
+++ b/res/values-tl/strings.xml
@@ -64,7 +64,7 @@
     <string name="uninstall_application_title" msgid="1860074100811653963">"I-uninstall ang app"</string>
     <string name="uninstall_update_title" msgid="4146940097553335390">"I-uninstall ang update"</string>
     <string name="uninstall_activity_text" msgid="6680688689803932550">"Bahagi ang <xliff:g id="ACTIVITY_NAME">%1$s</xliff:g> ng sumusunod na app:"</string>
-    <string name="uninstall_application_text" msgid="6691975835951187030">"Nais mo bang i-uninstall ang app na ito?"</string>
+    <string name="uninstall_application_text" msgid="6691975835951187030">"Gusto mo bang i-uninstall ang app na ito?"</string>
     <string name="uninstall_application_text_all_users" msgid="5574704453233525222">"Nais mo bang i-uninstall ang app na ito para sa "<b>"lahat"</b>" ng user? Aalisin ang application at ang data nito mula sa "<b>"lahat"</b>" ng user sa device."</string>
     <string name="uninstall_application_text_user" msgid="8766882355635485733">"Gusto mo bang i-uninstall ang app na ito para sa user na si <xliff:g id="USERNAME">%1$s</xliff:g>?"</string>
     <string name="uninstall_update_text" msgid="1394549691152728409">"Gusto mo bang palitan ang app na ito ng factory na bersyon? Maaalis ang lahat ng data."</string>
diff --git a/res/values-uk/strings.xml b/res/values-uk/strings.xml
index 5f42de8b..f9018706 100644
--- a/res/values-uk/strings.xml
+++ b/res/values-uk/strings.xml
@@ -64,7 +64,7 @@
     <string name="uninstall_application_title" msgid="1860074100811653963">"Видалити програму"</string>
     <string name="uninstall_update_title" msgid="4146940097553335390">"Видалити оновлення"</string>
     <string name="uninstall_activity_text" msgid="6680688689803932550">"Дія <xliff:g id="ACTIVITY_NAME">%1$s</xliff:g> є частиною такої програми:"</string>
-    <string name="uninstall_application_text" msgid="6691975835951187030">"Видалити цю програму?"</string>
+    <string name="uninstall_application_text" msgid="6691975835951187030">"Видалити додаток?"</string>
     <string name="uninstall_application_text_all_users" msgid="5574704453233525222">"Хочете видалити цю програму для "<b>"всіх"</b>" користувачів? Програму та її дані буде видалено для "<b>"всіх"</b>" користувачів цього пристрою."</string>
     <string name="uninstall_application_text_user" msgid="8766882355635485733">"Видалити цей додаток для користувача <xliff:g id="USERNAME">%1$s</xliff:g>?"</string>
     <string name="uninstall_update_text" msgid="1394549691152728409">"Відновити заводську версію цього додатка? Усі дані буде видалено."</string>
-- 
2.17.1

